partcad/src/partcad/converters/stl_converter.py

```
@@ -0,0 +1,100 @@
+from pathlib import Path
```

Ruff (0.8.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)